modules/ROOT/pages/mongodb/mongodb-connector-reference.adoc

```diff
@@ -14,2281 +18,2417 @@ MongoDB is an open source, high-performance, schema-free, document-oriented data
-| Collection Name a| String |  +++the name of the collection to create+++ |  | x
-| Capped a| Boolean |  +++if the collection will be capped+++ |  +++false+++ |
-| Max Objects a| Number |  +++the maximum number of documents the new collection is able to contain+++ |  |
-| Collection Size a| Number |  +++the maximum size of the new collection+++ |  |
+| Collection Name a| String |  +++The name of the collection to create+++ |  | x
+| Capped a| Boolean |  +++If the collection will be capped+++ |  +++false+++ | 
+| Max Objects a| Number |  +++The maximum number of documents the new collection is able to contain+++ |  | 
+| Collection Size a| Number |  +++The maximum size of the new collection+++ |  | 
 | Reconnection Strategy a| * <<reconnect>>
-* <<reconnect-forever>> |  +++A retry strategy in case of connectivity errors+++ |  |
+* <<reconnect-forever>> |  +++A retry strategy in case of connectivity errors.+++ |  | 
 |===
 
 
 === For Configurations
-
-* <<config>>
-
-=== Throws
-
-* MONGO:WRITE_CONCERN
-* MONGO:RETRY_EXHAUSTED
-* MONGO:UNKNOWN
-* MONGO:SERVER_ERROR
-* MONGO:BULK_WRITE
-* MONGO:IO_EXCEPTION
-* MONGO:SOCKET_READ_ERROR
-* MONGO:WRITE
-* MONGO:CLIENT_EXCEPTION
-* MONGO:QUERY_ERROR
-* MONGO:NODE_IS_RECOVERING
-* MONGO:MONGO_GRID_FS
-* MONGO:INVALID_CREDENTIALS
-* MONGO:EXECUTION_TIMEOUT
-* MONGO:SOCKET
-* MONGO:COMMAND_ERROR
-* MONGO:MongoExecution
-* MONGO:SOCKET_CLOSED
-* MONGO:SECURITY
-* MONGO:WAIT_QUEUE_FULL
-* MONGO:CURSOR_NOT_FOUND
-* MONGO:SOCKET_OPEN
-* MONGO:INTERNAL_EXCEPTION
-* MONGO:INTERRUPTED
-* MONGO:SOCKET_WRITE_ERROR
-* MONGO:CONNECTIVITY
-* MONGO:NOT_PRIMARY
-* MONGO:TIMEOUT
-* MONGO:DUPLICATE_KEY
-* MONGO:INCOMPATIBLE_DRIVER
-* MONGO:SOCKET_READ_TIMEOUT
-* MONGO:MAPPING_EXCEPTION
-* MONGO:ILLEGAL_ARGUMENT
+* <<config>> 
+
+==== Throws
+* MONGO:RETRY_EXHAUSTED 
+* MONGO:TIMEOUT 
+* MONGO:WAIT_QUEUE_FULL 
+* MONGO:COMMAND_ERROR 
+* MONGO:SOCKET 
+* MONGO:SOCKET_WRITE_ERROR 
+* MONGO:SOCKET_READ_TIMEOUT 
+* MONGO:INTERRUPTED 
+* MONGO:NOT_PRIMARY 
+* MONGO:WRITE 
+* MONGO:SOCKET_CLOSED 
+* MONGO:UNKNOWN 
+* MONGO:CURSOR_NOT_FOUND 
+* MONGO:INVALID_CREDENTIALS 
+* MONGO:ILLEGAL_ARGUMENT 
+* MONGO:WRITE_CONCERN 
+* MONGO:CLIENT_EXCEPTION 
+* MONGO:IO_EXCEPTION 
+* MONGO:SOCKET_READ_ERROR 
+* MONGO:QUERY_ERROR 
+* MONGO:MONGO_GRID_FS 
+* MONGO:SOCKET_OPEN 
+* MONGO:CONNECTIVITY 
+* MONGO:EXECUTION_TIMEOUT 
+* MONGO:SECURITY 
+* MONGO:MAPPING_EXCEPTION 
+* MONGO:NODE_IS_RECOVERING 
+* MONGO:INTERNAL_EXCEPTION 
+* MONGO:SERVER_ERROR 
+* MONGO:DUPLICATE_KEY 
+* MONGO:BULK_WRITE 
+* MONGO:INCOMPATIBLE_DRIVER 
+* MONGO:MongoExecution 
 
 
 [[createFileFromPayload]]
-== Create File From Payload
+=== Create File From Payload
 `<mongo:create-file-from-payload>`
 
 +++
-Creates a new MuleGridFSFile in the database, saving the given content, filename, contentType, and extraData, and answers it.
+Creates a new MuleGridFSFile in the database, saving the given content, filename, contentType, and extraData, and answers. it.
```

Comment:
I don't understand this. Answers what? Maybe it should be "answers in it." meaning that some sort of answers are part of the created file? Or maybe it answers the call to create the file by sending back a copy of the file, or a link to it?

Reply:
Good catch. I removed "it." I think the idea is that the content that's saved includes answers to some inquiry.

Reply:
OK, sounds good.